src/main/java/com/bunq/sdk/model/generated/Customer.java

```diff
@@ -0,0 +1,161 @@
+package com.bunq.sdk.model.generated;
+
+import com.bunq.sdk.context.ApiContext;
+import com.bunq.sdk.http.ApiClient;
+import com.bunq.sdk.http.BunqResponse;
+import com.bunq.sdk.http.BunqResponseRaw;
+import com.bunq.sdk.model.BunqModel;
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Used to view a customer.
+ */
+public class Customer extends BunqModel {
+
+  /**
+   * Field constants.
+   */
+  public static final String FIELD_BILLING_ACCOUNT_ID = "billing_account_id";
+
+  /**
+   * Endpoint constants.
+   */
+  private static final String ENDPOINT_URL_LISTING = "user/%s/customer";
+  private static final String ENDPOINT_URL_READ = "user/%s/customer/%s";
+  private static final String ENDPOINT_URL_UPDATE = "user/%s/customer/%s";
+
+  /**
+   * Object type.
+   */
+  private static final String OBJECT_TYPE = "Customer";
+
+  /**
+   * The id of the customer.
+   */
+  @Expose
+  @SerializedName("id")
+  private Integer id;
+
+  /**
+   * The timestamp of the customer object's creation.
+   */
+  @Expose
+  @SerializedName("created")
+  private String created;
+
+  /**
+   * The timestamp of the customer object's last update.
+   */
+  @Expose
+  @SerializedName("updated")
+  private String updated;
+
+  /**
+   * The primary billing account account's id.
+   */
+  @Expose
+  @SerializedName("billing_account_id")
+  private String billingAccountId;
+
+  public static BunqResponse<List<Customer>> list(ApiContext apiContext, Integer userId) {
+    return list(apiContext, userId, new HashMap<>());
+  }
+
+  public static BunqResponse<List<Customer>> list(ApiContext apiContext, Integer userId,
+      Map<String, String> params) {
+    return list(apiContext, userId, params, new HashMap<>());
+  }
+
+  /**
+   */
```

Comment:
Why do some of the methods have an empty doc comment and others don't ?

Reply:
Generated code has them in some cases. It can be addressed as an enhancement but not so important.

Reply:
generated